examples/piecestore/rpc/server/main.go

```diff
@@ -28,8 +28,13 @@ func main() {
 		}
 	}
 
-	dataFolder := "./"
+	// Get default folder for storing data and database
+	dataFolder, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
oh lol, you didn't need to do this. `./` was fine. i more meant having `.` be a flag itself. you don't need to change anything else here